ui-components/src/headers/Hero.tsx

```diff
@@ -29,7 +30,7 @@ const HeroButton = (props: { title: string; href: string; className?: string })
 )
 
 const Hero = (props: HeroProps) => {
-  let subHeader, styles
+  let subHeader, styles, secondaryTitle
```

Comment:
Can't we just use the subHeader for this instead of adding a new block to `Hero.tsx`?

Reply:
I thought we might still want to be able to show both sets of messages here, but if we feel Detroit won't show this warning (or a revised version around no open housing), we can just leverage subHeader.

![image](https://user-images.githubusercontent.com/82653098/128742108-7a159299-0d7a-4965-a547-ff5f43156264.png)

Reply:
Maybe a question for Tom?

Reply:
@tomgebauer Do you think we would ever want to show both messages above or just one?

Reply:
Speaking personally and trying to understand the original designer's intent - I prefer what's currently in the Hero component, where if we display the "applications closed" message, we hide the other message.

Reply:
I updated the Hero.tsx logic to have a default subHeader with our longer subtitle message. This gets overwritten with the existing logic for no applications or listings.

Reply:
Thanks, I meant have it as a child view to avoid changing the `Hero` component at all since that component already shows the child message as subtitle; i.e. instantiate it like:
```
<Hero ...>
  t(welcome message)
</Hero>
```